internal/git/infer_repo.go

```diff
@@ -0,0 +1,40 @@
+package git
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/sourcegraph/lsif-go/internal/command"
+)
+
+// InferRepo gets a Sourcegraph-friendly repo name from the git clone enclosing
```

Comment:
i think the comment here should say it's parsing it into lsif-go's expected format for packgeinformation rather than "sourcegraph-friendly"